t/13-metadata-multiple-signing.t

```diff
@@ -0,0 +1,43 @@
+use Test::Lib;
+use Test::Net::SAML2;
+use Net::SAML2::IdP;
+
+my $xml = path('t/idp-metadata-multiple-signing.xml')->slurp;
```

Comment:
I'm a bit of a fan to put testing data in a t/data dir, so everything is at one place.